pkgs/development/python-modules/lektor/default.nix

```
@@ -37,6 +38,8 @@ buildPythonPackage rec {
     flask pyopenssl ndg-httpsclient
```

setuptools is available during build/installation. This needs to be exposed at runtime
```suggestion
    flask pyopenssl ndg-httpsclient setuptools
```

---

Changed with https://github.com/NixOS/nixpkgs/compare/e4abda30967e1d949ce403613af83f6ef0566feb..07ba44d1b689c17c486e758002231d1c5dbf9f50